recipe-server/normandy/base/views.py

```diff
@@ -1,5 +1,9 @@
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpResponseRedirect
 
 
 def index(request):
     return HttpResponse('<h1>Normandy</h1>')
+
+
+def favicon(request):
+    return HttpResponseRedirect('/static/img/favicon.ico')
```

Comment:
@mythmon Templates use `STATIC_URL` variable for url, not the `static` tag. Shoul I use `settings.STATIC_URL` instead of `static` tag in view too?